### PR TITLE
security(deps): upgrade to fix audit

### PR DESCRIPTION
**Pull request by AviVahl**
### Description

`npm audit` on `master`:
found 33 vulnerabilities (28 moderate, 5 high) in 1353 scanned packages

`npm audit` on branch:
found 0 vulnerabilities in 1415 scanned packages

### CODE REVIEW

**Review by bkendall — APPROVED** [review]
Looks alright to me. Thanks!

**Comment by AviVahl** [comment]
I'm guessing the firebase upgrade broke the storage-emulator integration test. I'm trying the debug locally, but am not sure of root cause yet. I've ensured the same firebase version is loaded in the page, but issue still persists.

File: package.json [hunk 1]
```diff
@@ -107,7 +107,7 @@
     "filesize": "^6.1.0",
     "fs-extra": "^5.0.0",
     "glob": "^7.1.2",
-    "google-auth-library": "^6.1.3",
+    "google-auth-library": "^7.9.1",
```
[the remark this answers is not in the document]
Comment (AviVahl) [reply package.json:110]: https://github.com/googleapis/google-auth-library-nodejs/blob/main/CHANGELOG.md#700-2021-02-08

I tried ensuring everything uses v7, as I saw an error saying it does not detect a project id in the `it(...)` of the storage integration tests. The stack trace revealed two versions of auth-library, so ensured latest V7 is used across the board.

Didn't help. it seems start:emulators thinks port 4000 is already used, unlike the current `master`. It falls back to 4001. Not sure if this has anything to do with the failures, but my intuition tells me it does.

**Comment by AviVahl** [comment]
Would be great if anyone from the firebase team could take a look. I'm not familiar with the codebase enough to be able to debug this properly. :/

**Comment by AviVahl** [comment]
closing in favor of #3757

**Pull request closed without merging**